### ISSUE TITLE
Patrice's Task (11/06/23 - 11/17/2023)

### ISSUE THREAD

**Issue by natt96z** [natt96z/cybersac#132]
*Work on program script for competition interaction
*Visit/Work with MSA-Santa Ana Preparing Students Thursday @12:30PM (@userliluzibert @natt96z)
*Visit Valley High School Thursday (@computerbluerain )
*Visit Valley High School for Mentorship (OdyTro)
*Friday Zoom Meeting @ 3:00 PM (Team Meet Up)
*CC Emails/Communication Over SAC CyberPatriots Updates
*Mock Virtual Machines Review (Ubuntu is WIP)
*Add reader API to CyberSAC (WIP)
*Ask for student feedback (In-Person)
---------------------------------------------------------------------------------------------------------------------------------------
Please Note: Thursday's meetings at MSA and Valley are the main focus this week and are probably going to be a short one, 30 minutes to an hour mostly so work carefully and organized.

**Comment by natt96z** [comment]
![image](https://github.com/natt96z/cybersac/assets/112590005/0b96e861-7069-458e-9f11-d531d5ba574a)
![image](https://github.com/natt96z/cybersac/assets/112590005/ce2d1a82-cf35-4b16-b39e-397f8878b8d4)
![image](https://github.com/natt96z/cybersac/assets/112590005/8cf8778b-bc8a-4434-94f3-5854c616329f)
![image](https://github.com/natt96z/cybersac/assets/112590005/f3c0cd51-1ea0-41eb-9cc6-fa71ccf31fae)

10/31/2023: Me and @userliluzibert  finished setting up the Virtual Machine and just need to finish the scoring program setup for the OS. Overall most of the CyberSAC Windows stuff is completed, onto making a suitable Ubuntu Linux virtual machine. This same day Juarez emailed me back mentioning he's completed the registration process of the chosen students, so it looks like this Thursday round 2 is our for the taking. @computerbluerain helped fix the cisco packet tracer application on our VM as well thanks to her fixed power-point.

**Comment by natt96z** [comment]
![image](https://github.com/natt96z/cybersac/assets/112590005/b3e14201-a67e-466d-906d-662b597b6f5d)

**_11/13/2023 - Ubuntu Linux VM Final Touches and Zoom Meeting Over Discord Plans_**
The teamI recognized the importance of providing students with hands-on experience in a setting that mirrors the challenges they might encounter in real-world scenarios. To achieve this, we meticulously designed and implemented a virtual machine that replicates the intricacies of CyberPatriots competitions. our Ubuntu VM is well crafted and sharable via USB, scripting still needs to be added. @computerbluerain Is currently working on a quick and easy Cisco tutorial.

**Comment by natt96z** [comment]
![image](https://github.com/natt96z/cybersac/assets/112590005/78d83701-f10a-40e1-adce-e7026807e893)
![image](https://github.com/natt96z/cybersac/assets/112590005/368ae735-fb64-4c9e-bff3-5ad88e1c9214)


**_11/14/2023 - Follow Up With MSA, Recieval and Observation of Practice Round Material_**
 Our team recently gathered to review the newly introduced Practice Round 2 material collectively. We dedicated time to thoroughly examine and brainstorm strategies that would optimize our approach. Additionally, in preparation for our upcoming meeting with MSA/Valley High Schools on Thursday, we discussed key points to address and ensure a productive collaboration. @natt96z  It's worth noting that Mr. Juarez from MSA conveyed uncertainty regarding student participation after the RSVP email was sent. More information from Juarez will be provided shortly.
